cpp/src/gandiva/llvm_generator.cc

```diff
@@ -170,6 +170,9 @@ llvm::Value* LLVMGenerator::GetDataReference(llvm::Value* arg_addrs, int idx,
   llvm::Value* load = LoadVectorAtIndex(arg_addrs, idx, name);
```

Comment:
Not sure. But should we add this null type to supported types so that dremio can see this null type as a supported type. there seems to be some code in llvm_types.h for supported types. But I do not know this well enough to know whether you have to add something to supported types, so that dremio knows that Gandiva can be used as a viable codegen for this.

Reply:
Thanks @rkavanap. Could you involve some guy familiar with dremio and help review this? If dremio might update code correspondingly, I think the change should be small.